route-parser/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for route-parser 0.0
+// Project: https://github.com/rcs/route-parser
+// Definitions by: Ian Ker-Seymer <https://github.com/ianks>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'route-parser' {
+  class Route {
```

Comment:
I do not think this will work since I am only exporting the class. 

Maybe export = Route could work though. I tried but importing it failed in the the test.